requirements.txt

```diff
@@ -1,2 +1,3 @@
-premailer==3.6.2
-sendgrid<v6.0.0
+# generated from manifests external_dependencies
+premailer
```

Comment:
https://github.com/OCA/social/commit/36703f5d4b0ce45c585c6eefdb94279ff6de388e

Reply:
ah, in this case it must be set on the manifest of the module

Reply:
and if you look on pypi you see that https://pypi.org/project/premailer/3.6.2/ is the last one supporting py2.
The problem on runboat seems to be related to this too (it installs 3.10)